deploy/terraform-datacommons-website/modules/bt_cache/main.tf

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+locals {
+    resource_suffix = (
+        var.resource_suffix != null ?
+        format("-%s", var.resource_suffix) : ""
+    )
+}
+
+# BigTable instance used to store cache tables.
+resource "google_bigtable_instance" "bt_cache" {
+  name           = format("dc-graph%s", local.resource_suffix)
+  project        = var.project_id
+
+  cluster {
+    # There will be one cluster. Constant seems appropriate for now.
+    cluster_id   = "dc-graph-c1"
+    zone         = var.bt_instance_zone
+
+    autoscaling_config {
```

Comment:
@shifucun I just realized that we don't actually autoscale our BT instance right now (scaling up and down is signaled in GCF). For custom DC, do we want to keep existing behaviour(manual scaling)?

Reply:
For custom dc we should do autoscaling, can we update that?

Reply:
Current PR assumes autoscaling. I will update the gcf function with a follow up PR to remove the manual scaling.